x-pack/packages/kbn-elastic-assistant/impl/assistant/index.tsx

```diff
@@ -98,6 +99,11 @@ const AssistantComponent: React.FC<Props> = ({
     [conversationId, conversations, createConversation, selectedConversationId]
   );
 
+  // Remember last selection for reuse after keyboard shortcut is pressed
+  useEffect(() => {
+    setLastConversationId(selectedConversationId);
+  }, [selectedConversationId, setLastConversationId]);
+
```

Comment:
Might be a bug here if there's a `lastConversationId` set and all connectors have been deleted (in which case the UI should be locked to the `Welcome` conversation), so we should test a little around that scenario

Reply:
I think I will add an extra cleanup step somewhere around the connector deletion then

Reply:
Added an extra cleanup whenever connector list is fetched & it is empty:)